archinstall/__init__.py

```diff
@@ -33,6 +33,8 @@ def initialize_arguments():
 	parser.add_argument("--creds", nargs="?", help="JSON credentials configuration file")
 	parser.add_argument("--silent", action="store_true",
 						help="Warning!!! No prompts, ignored if config is not passed")
```

Comment:
'Warning!!!' also looks out of place here now that I think about it. Let's make this help text something like "WARNING: Disables all prompts for input. If no configuration is provided, this is ignored"